drivers/clock_control/Kconfig

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if BT_LL_NRFXLIB
+
+config CLOCK_CONTROL_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller clock control driver"
+	default y
```

Comment:
"default y" is bad practice.

Is it possible to avoid in this case?

Reply:
Only using `select` from the `BT_LL_NRFXLIB` as far as I can tell. I'd like to avoid using `select`, since it is technically possible to run the controller without the clock control (glue) driver.

Reply:
Shouldn't there be some kind of dependency between CLOCK_CONTROL and CLOCK_CONTROL_NRF_LL_NRFXLIB ?

Reply:
If CLOCK_CONTROL_NRF_LL_NRFXLIB depends on CLOCK_CONTROL will it be implicitly selected?

Reply:
Never mind, I see that the relationship between HAS_CLOCK_CONTROL, CLOCK_CONTROL, and CLOCK_CONTROL_IMPLEMENTATION_X is already broken, so no dependency is needed.